### PR TITLE
Speed up GeoJSON export

### PR DESCRIPTION
**Pull request by velvia**
- using the Grisu Double toString serializer
- using optimizations in third-party-utils GeoJSON encoder

GeoJSON export should be roughly an order of magnitude faster.

### CODE REVIEW

**Comment by urmilan** [comment]
Can you investigate the [test failure](http://jenkins.aws-us-west-2-infrastructure.socrata.net/job/sbt-build/530/console)? Apart from that lgtm.

File: project/Dependencies.scala [hunk 1]
```diff
@@ -17,13 +17,14 @@ object Dependencies {
     val postgresql        = "9.1-901-1.jdbc4"
     val quasiQuotes       = "2.0.0"
     val rojomaJson        = "3.2.2"
```
Comment (urmilan) [inline project/Dependencies.scala]: Do you still need this?

Comment (velvia) [inline project/Dependencies.scala]: Not really but might be good to keep around to easily switch back.

**Comment by sokrahta** [comment]
lgtm :shipit:

**Pull request merged**